cardano-api/src/Cardano/Api/ProtocolParameters.hs

```diff
@@ -169,7 +183,7 @@ data ProtocolParameters =
        -- | The minimum permitted value for new UTxO entries, ie for
        -- transaction outputs.
        --
-       protocolParamMinUTxOValue :: Lovelace,
+       protocolParamMinUTxOValue :: Maybe Lovelace,
```

Comment:
If this became unused in Alonzo, we should add a comment to that effect here too, that it is used in Shelley to Mary and not after.